src/main/java/com/gregtechceu/gtceu/data/lang/MaterialLangGenerator.java

```diff
@@ -2,14 +2,20 @@
 
 import com.gregtechceu.gtceu.api.data.chemical.material.Material;
 import com.gregtechceu.gtceu.api.data.chemical.material.registry.MaterialRegistry;
+import com.gregtechceu.gtceu.common.data.GTMaterials;
 import com.tterrag.registrate.providers.RegistrateLangProvider;
 
 import static com.gregtechceu.gtceu.utils.FormattingUtil.toEnglishName;
 
 public class MaterialLangGenerator {
 
     public static void generate(RegistrateLangProvider provider, MaterialRegistry registry) {
-        for (Material material : registry.getAllMaterials())
-            provider.add(material.getUnlocalizedName(), toEnglishName(material.getName()));
+        for (Material material : registry.getAllMaterials()) {
+            if (material == GTMaterials.Iron3Chloride) {
+                provider.add(material.getUnlocalizedName(), "Iron III Chloride");
+            } else {
+                provider.add(material.getUnlocalizedName(), toEnglishName(material.getName()));
+            }
+        }
```

Comment:
wrong place, put it in `LangHandler` around the other replacements.

Reply:
moved, MaterialLangGenerator has been reverted